docs/userguide/containers/dockerrepos.md

```diff
@@ -42,6 +42,7 @@ You can now commit and push your own images up to your repos on Docker Hub.
 > **Note:**
 > Your authentication credentials will be stored in the `~/.docker/config.json`
 > authentication file in your home directory.
+> For Windows users, this Docker file is stored in `$HOME/.docker/config.json`
```

Comment:
Before you try a command, if you haven't already, create a Docker ID. You can do
this directly on [Docker Hub](https://hub.docker.com/). Once you have a Docker ID you can login into your account directly from the Engine CLI. 

```bash
$ docker login
```

The `login` command stores your authentication credentials in the $HOME/.docker/config.json if you are in a Bash shell. For Windows `cmd` users this file is in `%HOME%\.docker\config.json`, while for PowerShell users the location is designated as `$env:Home\.docker\config.json`.

Once you have logged in, you can commit and push images from the command line to
your repos on Docker Hub.